sphynx/lynxkite-sphynx/vertex_set_intersection.go

```diff
@@ -4,49 +4,21 @@ package main
 
 import (
 	"fmt"
-	"sort"
-	"sync"
 )
 
 type MergeVertexEntry struct {
 	id    int64
 	count int
 }
-type MergeVertexEntrySlice []MergeVertexEntry
-
-func (a MergeVertexEntrySlice) Len() int {
-	return len(a)
-}
-func (a MergeVertexEntrySlice) Less(i, j int) bool {
-	return a[i].id < a[j].id
-}
-func (a MergeVertexEntrySlice) Swap(i, j int) {
-	a[i], a[j] = a[j], a[i]
-}
-
-type Int64Slice []int64
-
-func (a Int64Slice) Len() int {
-	return len(a)
-}
-func (a Int64Slice) Less(i, j int) bool {
-	return a[i] < a[j]
-}
-func (a Int64Slice) Swap(i, j int) {
-	a[i], a[j] = a[j], a[i]
-}
 
 func doVertexSetIntersection(vertexSets []*VertexSet) (intersection *VertexSet, firstEmbedding *EdgeBundle) {
-	mergeVertices := make(MergeVertexEntrySlice, len(vertexSets[0].MappingToUnordered))
+	mergeVertices := make([]MergeVertexEntry, len(vertexSets[0].MappingToUnordered))
```

Comment:
We could avoid this map as well via a multi-merge in a single nested loop. But no need to it in this PR.